src/main/scala/org/broadinstitute/hail/driver/ExportVCF.scala

```diff
@@ -205,7 +205,57 @@ object ExportVCF extends Command {
           sb.append("GT:AD:DP:GQ:PL")
         gs.foreach { g =>
           sb += '\t'
-          sb.append(g)
+
+          sb.append(g.gt.map { gt =>
+            val p = Genotype.gtPair(gt)
+            s"${ p.j }/${ p.k }"
+          }.getOrElse("./."))
+
+          def appendOption(stringBuilder: StringBuilder, toAppend: Option[Array[Int]]): Unit = {
+            if (toAppend.isDefined)
+              toAppend.foreach(_.foreachBetween(sb.append(_))(sb += ','))
+            else
+              sb += '.'
```

Comment:
You have `stringBuilder` argument but still reference `sb`.  I'd move these to the ExportVCF object level so there's no chance of shadowing.